app/views/users/show_follow.html.erb

```diff
@@ -0,0 +1,30 @@
+<% provide :title, @title %>
+<div class="row">
+  <aside class="col-md-4">
+    <section class="user_info">
+      <%= gravatar_for @user %>
+      <h1><%= @user.name %></h1>
+      <span><%= link_to "view my profile", @user %></span>
+      <span><b>Microposts:</b> <%= @user.microposts.count %></span>
```

Comment:
https://viblo.asia/p/i18n-in-rails-part-2-GrLZDAO3lk0
Nghiên cứu đưa chỗ này vào I18n sử dụng hiển thị cho số nhiều ddie m